### DISCUSSION TITLE
Extend current Category Blog, List Views to Work with Just Tags

### DISCUSSION THREAD

**Discussion by parthlawate** [joomla/joomla-cms#31549] [2. Cold Features]
### Problem identified
Right now there is no easy way to create a Blog view of Articles via com_content showing content only for one or more tags. The views available vi com_tags arent as well maintained or as rich.. 

### Proposed solution
Right now the Category blog view already allows choosing specific tags . This view /menu can be renamed to just a Blog view and allow selection of just tags. Currently choosing a category is compulsory. If we remove this compulsion the view can become more flexible and can be used with just tags or as a combination as it works now. 

### Open questions
NA
![Screenshot 2020-12-01 at 4 38 25 PM](https://user-images.githubusercontent.com/673176/100736771-3a204a00-33f9-11eb-944d-0d5a103bfdd9.png)
![Screenshot 2020-12-01 at 4 38 19 PM](https://user-images.githubusercontent.com/673176/100736780-3d1b3a80-33f9-11eb-961d-58002225581d.png)

**Comment by B3nito** [comment]
Hello parthlawate! But you can easy create a alternate menutype for your own needing for this? Or you use tag menu type

**Comment by parthlawate** [comment]
You mean code it in right ? We already do that.. However this can be a good out of the box solution for a very standard use case. The tag menu type is very restrictive and not as maintained as the Article Menu types.. 

Most of joomla's direct user wont code custom menus/ layouts. 

Tagging is more flexible in many scenarios (An article can be part of multiple tags for instance)  & you can end up using tags only instead of categories if this is enabled.

**Reply by brianteeman** [reply]
If you already do it why do you keep it to yourselves? Submit a pull request

**Reply by parthlawate** [reply]
I see the way we do it as inefficient right now ..  This comment was against the earlier comment of creating an alternate MVC.. 

So I wanted to see if we can agree on an approach before creating a PR. My Proposed approach is to have each 'View' type menu (Blog/Table/Card views) support filters for - Categories, Tags , Authors etc.  Right now the category based menus are the most powerful.. Extending those to support additional filters i the approach i prefer. That is what i have proposed & I am looking for comments against that .. I think thats the point of the RFC ? get more thoughts, ideas & alignment before committing coding time to it ?

**Comment by ssnobben** [comment]
**Tagging is more flexible in many scenarios (An article can be part of multiple tags for instance) & you can end up using tags only instead of categories if this is enabled.** Exactly and you then can have dynamic filters where you also can sort items multidimensional/hierarchically
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31549">issues.joomla.org/tracker/joomla-cms/31549</a>.</sub>

**Comment by leoalv** [comment]
From my point of view it is well how it is working now, but what you propose is a good feature, and I think it would be best to create a new View and Model, it is the same thing that I am requesting to show articles of a selected author

**Comment by parthlawate** [comment]
One approach is create new Menu types for each type of Filter that you might need - Categories, Tags , Authors OR create one menu type that allows us to one one or more filters
